muselsl/muse.py

```diff
@@ -238,16 +238,15 @@ def select_preset(self, preset=21):
 
         See details on https://goo.gl/FPN1ib
         For 2016 headband, possible choice are 'p20' and 'p21'.
-        Untested but possible values are 'p22' and 'p23'
+        Untested but possible values include 'p22','p23','p31','p32','p50','p51','p52','p53','p60','p61','p63','pAB','pAD'
```

Comment:
Presets docs missing (link invalid).  Best alternative I currently found is here: https://github.com/urish/muse-js/issues/42#issuecomment-743689736

Reply:
there might be more information near the "muse mind monitor" application and muse's old "muselab" and research sdk software.  they used to actually document the presets, but i'm not sure where to find that documentation.  could you open a pull request with any helpful links you find, so that others can find your work in years to come, to add on to it?